test/pytest/test_zeropadding_pytorch_onnx.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+
+import numpy as np
+import qonnx.util.cleanup
+import torch
+import torch.nn as nn
+from qonnx.core.modelwrapper import ModelWrapper
+
+from hls4ml.converters import convert_from_onnx_model, convert_from_pytorch_model
+from hls4ml.utils.config import config_from_onnx_model, config_from_pytorch_model
+
+test_root_path = Path(__file__).parent
+
+
+def test_constantpad_1d():
+    class Pad1DModel(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.pad = nn.ConstantPad1d((2, 3), 0)  # pad 2 left, 3 right
+
+        def forward(self, x):
+            return self.pad(x)
+
+    model = Pad1DModel()
+    model.eval()
+    config_pytorch = config_from_pytorch_model(model, (2, 4), channels_last_conversion='off')
+    hls_model_pytorch = convert_from_pytorch_model(
+        model, output_dir=str(test_root_path / 'hls4mlprj_constpad_1d/pytorch'), hls_config=config_pytorch
+    )
+
+    hls_model_pytorch.compile()
+
+    onnx_path = str(test_root_path / 'hls4mlprj_constpad_1d/pad1d.onnx')
+    torch.onnx.export(model, torch.randn(1, 2, 4), onnx_path, opset_version=10)
```

Comment:
Isn't the now recommended usage now with `dynamo=True`? I have previously followed the recipe from https://docs.pytorch.org/tutorials/beginner/onnx/export_simple_model_to_onnx_tutorial.html successfully, though I can't guarantee that it would work here.

Reply:
I saw that the docs now push `dynamo=True` a lot, but wasn't sure what's our stance on it. I can change if that's what you prefer? I realize now that I have to change this line anyway, `opset_version=10` is a workaround for my local env, it should not be in the code.